{"uuid":"3a664997-3391-4a9f-94c5-ea1b21be0ec5","historyId":"4f4e9ddc71671fe9b96199eb36aee673","fullName":"com.example.pruebas.AdminSigninTest.IncioSesion","labels":[{"name":"package","value":"com.example.pruebas.AdminSigninTest"},{"name":"testClass","value":"com.example.pruebas.AdminSigninTest"},{"name":"testMethod","value":"IncioSesion"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"Pruebas"},{"name":"subSuite","value":"com.example.pruebas.AdminSigninTest"},{"name":"host","value":"MacBook-Pro-6.local"},{"name":"thread","value":"28497@MacBook-Pro-6.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"IncioSesion","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [La contrase침a es inv치lida] but found [Usuario no existente]","trace":"java.lang.AssertionError: expected [La contrase침a es inv치lida] but found [Usuario no existente]\n\tat org.testng.Assert.fail(Assert.java:110)\n\tat org.testng.Assert.failNotEquals(Assert.java:1577)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\n\tat org.testng.Assert.assertEquals(Assert.java:131)\n\tat org.testng.Assert.assertEquals(Assert.java:655)\n\tat org.testng.Assert.assertEquals(Assert.java:665)\n\tat com.example.pruebas.AdminSigninTest.IncioSesion(AdminSigninTest.java:72)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:578)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:829)\n\tat org.testng.TestRunner.run(TestRunner.java:602)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:437)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\n\tat org.testng.TestNG.run(TestNG.java:1067)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1686799468183,"stop":1686799475772}